src/Mod/PartDesign/App/FeatureSketchBased.cpp

```diff
@@ -404,7 +404,7 @@ TopoDS_Shape ProfileBased::getVerifiedFace(bool silent) const {
     return TopoDS_Face();
 }
 
-TopoShape ProfileBased::getProfileShape() const
+TopoShape ProfileBased::getProfileShape(bool needSubElement) const

```

Comment:
In that case I think we may pass on it, but in general we try to avoid boolean arguments like this. That is because on call-site and when reading code you are left with `getProfileShape(true)` and meaning of true is undefined. In such cases we recommend to use small enums with well named options.

Given however that code called by you has the same problem I don't think that you should do anything in that PR - this is simply refactor opportunity to go through all functions with `needSubElement` argument, make enum and make use of it.